### DISCUSSION TITLE
Header?

### DISCUSSION THREAD

**Discussion by themarkroberts** [holger1411/understrap#1426] [Q&A] [answered]
Why doesn't header.php have an actual header element?

**Comment by bacoords** [comment, marked as answer]
Interesting question and I hope to see some answers and opinions here.

As I've understood it, the `<header>` element was originally for the header of the content, not just the masthead or navigation of the site. We put `<header>` around the `<h1>` in the individual content templates, inside `<article>` elements, and around the `<h1>` on the archive template, but should it also be around the navigation? Maybe. 

It seems that [Underscores does](https://github.com/Automattic/_s/blob/master/header.php) use a `<header>` these days, so maybe that original definition has expanded. If anything, it should probably be a header element when it does display the .navbar-brand as an h1, and maybe it should just be changed globally
